src/web/common.rkt

```diff
@@ -54,7 +54,16 @@
     ("FPCore" . ,fpcore->string)
     ("C" . ,(curryr core->c "code"))))
 
-(define (render-program #:to [result #f] test)
+(define (render-preprocess-struct preprocess)
+  (define vars (string-append "[" (string-join (map symbol->string (symmetry-group-variables preprocess)) " ") "]"))
+  (render-large vars "=" (string-append "sort(" vars ")")))
```

Comment:
I think we should have some kind of font for `sort` like `\mathsf{sort}`. Otherwise it'll look weird.

Reply:
👍